<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatenIsland</class>
 <widget class="QDialog" name="StatenIsland">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="DisText8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText11">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>640</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText13">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>740</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Staten Island Locations</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>590</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText5">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>340</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText12">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>690</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText8">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>490</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText1">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText10">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>590</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>390</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>540</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText7">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>440</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>290</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText12">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>690</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText11">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>640</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText13">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>740</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>590</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText11">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>640</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>190</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText13">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>740</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DisText3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LocText12">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>690</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="NameText2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Rockwell&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Farmers' Market Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>90</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Farmers' Market Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>District</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Community</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>720</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(146, 144, 170);</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
